### PR TITLE
Create 2021-February-3.md

### PR DESCRIPTION
**Pull request by chriswhilar**

### CODE REVIEW

**Comment by rorezende** [comment]
LGTM

**Review by rorezende — APPROVED** [review]

File: releasenotes/2021-February-3.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+    identity: CommunicationUserIdentifier | PhoneNumberIdentifier | CallingApplicationIdentifier | MicrosoftTeamsUserIdentifier | UnknownIdentifier | undefined;
+}
+```
+
+4. Changed Call.isIncoming: boolean to Call.direction: CallDirection
+```js
+    export declare type CallDirection = 'Incoming' | 'Outgoing';
+```
+
+3. Type name changes from @azure/communication-common
```
Comment (akania) [inline releasenotes/2021-February-3.md:50]: nit, please mention that calling-beta.4 depends on common-beta.4

File: releasenotes/2021-February-3.md [hunk 2]
```diff
@@ -0,0 +1,192 @@
+This release notes contain new changes for ACS Calling Web (JavaScript) SDK v1.0.0-beta.4
+
+# Bug fixes
+1. Fix CallEndReason.subCode lower case 'c' typo 
+2. Fix CallAgent.join() api accepting undefined CallContext. Api must now take a GroupLocator or MeetingLocator 
+
+# Breaking API Changes
+1. New flow for collection subscriptions:
```
Comment (akania) [inline releasenotes/2021-February-3.md:8]: nit, do mention that application has to 
1) inspect existing values of properties & collections 
2) subscribe to property & collection update events

In order to get current & future values

File: releasenotes/2021-February-3.md [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+
+```js
+const callTransferApi = call.api(Features.Transfer);
+```
+
+Call transfer involves three parties *transferor*, *transferee*, and *transfer target*. Transfer flow is working as following:
+
+1. There is already a connected call between *transferor* and *transferee*
+2. *transferor* decide to transfer the call (*transferee* -> *transfer target*)
+3. *transferor* call `transfer` API
```
Comment (akania) [inline releasenotes/2021-February-3.md:96]: nit, Transferror should put current call on hold first, before invoking 'transfer' API

File: releasenotes/2021-February-3.md [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+        * sdk.bundle.js
+    * /types
+        * communication-calling.d.ts
+    * EULA.txt
+    * Notices.txt
+    * package.json (Updated)
+    * README.md (Updated)
+2. Added new package dependencies:
+    * @azure/logger
+    * @azure/communication-common
```
Comment (akania) [inline releasenotes/2021-February-3.md:191]: nit ! calling-beta.4 requires @azure/communication-common 1.0.0-beta.4

**Pull request merged**